PokeAlarm/Alarms/Telegram/TelegramAlarm.py

```diff
@@ -205,7 +208,9 @@ def generic_alert(self, alert, dts):
         message = replace(alert.message, dts)
         lat, lng = dts['lat'], dts['lng']
         max_attempts = alert.max_attempts
-        sticker_url = replace(alert.sticker_url, dts)
+        sticker_url = replace(alert.display_sticker_url, dts) \
+            if alert.use_display_sticker is False\
+            else replace(alert.sticker_url, dts)
```

Comment:
Is this section working as intended? From my reading, it is using `alert.display_sticker_url` if `alert.use_display_sticker` is `False`. Isn't it supposed to be the other way around?

Reply:
You are correct, I must have missed this